### PR TITLE
TNO-2267: Fix and update icons for evening overview

### PR DESCRIPTION
**Pull request by jdtoombs**
Trying to test - but ran into another environment issue. On draft until I fix and test again!

In this PR:

- update icons to match mock-up
- new migration 1.0.133
- fix bug of icons not displaying on report rows (for video and transcript available)

### CODE REVIEW

File: app/subscriber/src/components/header/env.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "dev": ["localhost", "tno-dev.apps.silver.devops.gov.bc.ca", "dev.mmi.gov.bc.ca"],
+  "dev": [ "tno-dev.apps.silver.devops.gov.bc.ca", "dev.mmi.gov.bc.ca"],
```
Comment (jdtoombs) [inline app/subscriber/src/components/header/env.json:2]: Oops this was just for me - will remove. Easier on my eyes :stuck_out_tongue:

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql [hunk 2]
```diff
@@ -0,0 +1,110 @@
+
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql:1]: Going to format this like the Weekday - will update

**Review by Fosol — COMMENTED** [review]
Can  you comment on what you changed?  These sql scripts are difficult to review as we can't compare with the prior version.

**Comment by jdtoombs** [comment]
@Fosol - will do, I will ping again when it is ready - just dealing with some environment bugs at the moment!

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+UPDATE public.report_template SET
+  "body" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using System
+@using System.Linq
+@using TNO.Entities
+
+<style>
+a { text-decoration:none; }
+.video-icon { color: #702828; }
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/00-WeekdayTemplateUpdate.sql:12]: new colours

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+UPDATE public.report_template SET
+  "body" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using System
+@using System.Linq
+@using TNO.Entities
+
+<style>
+a { text-decoration:none; }
+.video-icon { color: #702828; }
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql:12]: new colours for icons

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                };
+                var summary = String.IsNullOrEmpty(item.Summary) ? "---" : item.Summary.Replace("\n", "<br/>");
+                <tr style="border-bottom: 1px solid #DED9DA;">
+                    <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@placement</td>
+                    <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@(time?.Length == 8 ? time.Substring(0, 5) : time)</td>
+                    <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">
+                        @if (item.ContentId.HasValue)
+                        {
+                            <a href="@($"{ViewContentUrl}{item.ContentId}")">@summary</a>
+                            @if (item.Content.FileReferences.Count() > 0 && item.Content.FileReferences.First().ContentType.StartsWith("video/"))
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql:70]: this previously contained `.StartsWith("audio/")` now `video`

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/00-WeekdayTemplateUpdate.sql [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        };
+        var summary = String.IsNullOrEmpty(item.Summary) ? "---" : item.Summary.Replace("\n", "<br/>");
+        <tr style="border-bottom: 1px solid #DED9DA;">
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@placement</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@(time?.Length == 8 ? time.Substring(0, 5) : time)</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">
+            @if (item.ContentId.HasValue)
+            {
+              <a href="@($"{ViewContentUrl}{item.ContentId}")">@summary</a>
+              @if (item.Content.FileReferences.Count() > 0 && item.Content.FileReferences.First().ContentType.StartsWith("video/"))
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/00-WeekdayTemplateUpdate.sql:73]: this previously contained .StartsWith("audio/") now video

File: libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    var itemCount = section.Items.Count(i => i.ItemType == AVOverviewItemType.Story);
+
+    <div class="section" style="border: 1px solid #9F9196; border-radius: 6px; margin-bottom:20px; padding: 16px;width:1150px;">
+
+        <div class="section-header" style="text-align:center; font-size:1.1rem; font-weight:600; padding-bottom:6px;">
+            @section.Name @(section.StartTime?.Length == 8 ? section.StartTime.Substring(0, 5) : section.StartTime) @(section.StartTime?.Length > 0 ? "hr" : "")
+            @if (!String.IsNullOrEmpty(section.Anchors)) { <span> - @section.Anchors</span> }
+        </div>
+
+        <table style="width:100%; border-collapse: collapse;">
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.131/Up/PostUp/01-WeekendTemplateUpdate.sql:39]: Added indentation for readability

**Review by Fosol — APPROVED** [review]

**Pull request merged**